sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/ResourceConfiguration.java

```diff
@@ -31,13 +31,19 @@ static Resource configureResource(
       BiFunction<? super Resource, ConfigProperties, ? extends Resource> resourceCustomizer) {
     Resource result = Resource.getDefault();
 
+    Set<String> enabledProviders =
+        new HashSet<>(config.getList("otel.java.enabled.resource.providers"));
```

Comment:
Should we validate that only one of these is set as it would probably result in confusing behavior?

Reply:
I think the only missing case should be only setting the `otel.java.enabled.resource.providers` configuration. Only setting `` configuration is validated in other test case.